testdata/p4_16_samples/psa-i2e-cloning-basic-bmv2.p4

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2020-present Akamai Technologies, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+#include <core.p4>
+#include "psa.p4"
+
+
+typedef bit<48>  EthernetAddress;
+
+header ethernet_t {
+    EthernetAddress dstAddr;
+    EthernetAddress srcAddr;
+    bit<16>         etherType;
+}
+
+struct empty_metadata_t {
+}
+
+struct metadata_t {
+}
+
+struct headers_t {
+    ethernet_t       ethernet;
+}
+
+parser IngressParserImpl(packet_in pkt,
+                         out headers_t hdr,
+                         inout metadata_t user_meta,
+                         in psa_ingress_parser_input_metadata_t istd,
+                         in empty_metadata_t resubmit_meta,
+                         in empty_metadata_t recirculate_meta)
+{
+    state start {
+        pkt.extract(hdr.ethernet);
+        transition accept;
+    }
+}
+
+control cIngress(inout headers_t hdr,
+                 inout metadata_t user_meta,
+                 in    psa_ingress_input_metadata_t  istd,
+                 inout psa_ingress_output_metadata_t ostd)
+{
+    action clone() {
+        ostd.clone = true;
+        ostd.clone_session_id = (CloneSessionId_t) (CloneSessionIdUint_t) 3;
+    }
+
+    apply {
+        clone();
+
+        if (hdr.ethernet.dstAddr == 0) {
+            // This action should overwrite the ostd.drop field that
```

Comment:
This comment is probably copied and pasted from another PSA test program, and doesn't appear to be relevant for this test program, since there is no send_to_port() call above this comment.

Reply:
Ah yep my bad, fixing this in next commit.